main.py

```diff
@@ -0,0 +1,78 @@
+import json
+from tkinter import *
+from tkinter import messagebox
+data_file = open("data.json")
```

Comment:
One line

Reply:
unfortunately it won't let me include messagebox on the same line as * because it is a module.